ubuntu/ubuntu1710.json

```diff
@@ -0,0 +1,12 @@
+{
+  "_comment": "Build with `packer build -var-file=ubuntu1710.json ubuntu.json`",
+  "vm_name": "ubuntu1710",
+  "cpus": "1",
+  "disk_size": "65536",
+  "iso_checksum": "af913c00296eb0cfece99b8a02ae6db035ae173ed240241e780c8d7db96914b1",
+  "iso_checksum_type": "sha256",
+  "iso_url": "http://releases.ubuntu.com/artful/ubuntu-17.10-server-amd64.iso",
```

Comment:
url doesn't exist?

Reply:
Ah, yes - I've been running out of my cache all this time :-(